fiftyone/core/odm/database.py

```
@@ -939,6 +1015,32 @@ def list_datasets():
+
+    Args:
+        id (None): the ID of the dataset
+        name (None): the name of the dataset
+
+    Returns:
+        a :class:`fiftyone.core.dataset.Dataset`
+    """
+    if id is not None:
+        try:
+            uid = ObjectId(id)
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)